src/style.css

```diff
@@ -1,3 +1,76 @@
-body {
+* {
```

Comment:
Using the universal selector `*` to reset styles can have performance implications, especially on large documents. Consider using a more specific reset or a CSS reset library.